src/main/java/br/pucrs/ages/mase/plan_service/entity/PlanTemplate.java

```diff
@@ -5,10 +5,12 @@
 import org.springframework.data.annotation.Id;
 import org.springframework.data.mongodb.core.mapping.Document;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
 @Document(collection = "plantemplates")
 public class PlanTemplate {
 	
-	private @Id String id;
+	private @JsonIgnore @Id String id;
```

Comment:
Using @jsonIgnore here, will never send de planTemplate in the responsePayload, is that on purpose?

Reply:
Yes. And it is necessary for object mapping.

Reply:
Okay, and how the client (frontEnd, Android), will specify which plan they want, if they never receive their Id? Perhaps you should consider an output dto for this instance, because it seems kind of odd that your entity needs a jsonIgnore.

Reply:
He is supposed to load all of them. The choosing happens at the mobile level.

Reply:
Makes sense for our domain.